apps/betterangels-backend/clients/tests/test_queries.py

```diff
@@ -205,6 +205,50 @@ def test_client_profiles_query_search(
         client_profiles = response["data"]["clientProfiles"]
         self.assertEqual(len(client_profiles), expected_client_profile_count)
 
+    @parametrize(
+        ("full_name, expected_client_profile_count"),
+        [
+            ("", 0),  # no filters
+            ("Todd", 0),  # too few search fields
+            ("Todd,Gustav,Chavez", 0),  # too many search fields
+            ("Tod,Chavez", 0),  # inexact match on first name
+            ("Todd,Chave", 0),  # inexact match on last name
+            ("Todd,Chavez", 1),  # exact match on first & last name
+            ("Todd , Chavez", 1),  # exact match on first & last name
+            ("Todd Gustav,Chavez", 1),  # exact match on first & last name
```

Comment:
It's a bit confusing why 217 and 218 would both pass. 
I may be misunderstanding this, but can we guarantee that `Gustav` here is a middle name and not actually stored as `firstName`?

Reply:
@tglaz oh sorry i wasn't clear, "Todd Gustav" _is_ the first name (for new client created on line 228 below). this test is showing that this search doesn't use the middle name at all, but does support spaces in the first name. 
whereas the 2 lines above return the `self.client_profile_1 ` client

Reply:
so is the first name is `partial` match?

Reply:
no it's an exact match, just case insensitive @tglaz